src/index.js

```diff
@@ -24,13 +25,45 @@ const addTask = (e) => {
     attr.value = index;
     article.setAttributeNode(attr);
     article.innerHTML = `<div class="inp-p">
-    <input type="checkbox">
+    <input type="checkbox" class="checkb">
     <div class="todo-desc"><p class="todo-item">${task}</p></div>
 </div>
 <div class="icons">
     <i class="fa-solid fa-ellipsis-vertical"></i>
     <i class="fa-solid fa-trash-clock fa-trash"></i>
 </div>`;
```

Comment:
- [ ] [OPTIONAL] Please make sure that when the to-do items are populated in the page, they have the right "completed" state. Currently, when a todo is populated into the page from local storage, it shows as not completed (unchecked checkbox) whether it was actually completed or not.